src/api/test/test_helper.rb

```diff
@@ -170,7 +170,7 @@ def add_auth(headers)
         headers
       end
 
-      alias real_process process_with_kwargs
+      alias_method :real_process, :process_with_kwargs
```

Comment:
Not long ago rubocop complained about using alias_method forced us to use alias. Funny that it's now the opposite.